apps/docs/src/guide/abi-typegen/using-generated-types.md

```diff
@@ -40,6 +40,16 @@ const contract = await MyContract__factory.deployContract(bytecode, wallet);
 console.log(contract.id);
 ```
 
+You can also pass in [`DeployContractOptions`](https://github.com/FuelLabs/fuels-ts/blob/a64b67b9fb2d7f764ab9151a21d2266bf2df3643/packages/contract/src/contract-factory.ts#L19-L24) like storage slots and configurable constants to the `deployContract` method:
+
+```ts
+import storageSlots from "../contract/out/debug/storage-slots.json";
+
+const contract = await MyContract__factory.deployContract(bytecode, wallet, {
+  storageSlots,
+});
+```
+
```

Comment:
Can we move this to a docs-snippets test so that it does not get outdated?

Reply:
@danielbate agreed, but I also found out that the rest of the snippets in this file are hard-coded as well and have a todo attached to them about moving them to doc-snippets. Do you mind if I tackle all of them at once in a follow-up issue after this PR?

Reply:
Makes sense 👍🏻